coforall-throws-strict-error.chpl:4: In function 'test':
coforall-throws-strict-error.chpl:5: error: call to throwing function coforall_fn without try or try! (strict mode)
coforall-throws-strict-error.chpl:5: note: throwing function coforall_fn defined here
coforall-throws-strict-error.chpl:6: note: call to throwing function here
